TrackletAlgorithm/MatchEngine.cc

```diff
@@ -220,6 +220,9 @@ void MatchEngineTop(const BXType bx, BXType& bx_o,
 #pragma HLS interface register port=bx_o
 #pragma HLS resource variable=inputStubData->get_mem() latency=2
 #pragma HLS resource variable=inputProjectionData->get_mem() latency=2
+// For use with Vitis >=2020.1
```

Comment:
@aperloff Is it possible to automatically choose, using preprocessor macros, which set of pragmas is enabled? This document claims there is a predefined `__VITIS_HLS__` macro that might be useful:
https://www.xilinx.com/html_docs/xilinx2020_2/vitis_doc/migrating_to_vitis_hls.html

Reply:
Done